visualization/app/codeCharta/ui/attributeSideBar/ccNodePath/ccNodePath.component.ts

```diff
@@ -0,0 +1,39 @@
+import { IRootScopeService } from "angular"
+
+import { CodeMapBuilding } from "../../codeMap/rendering/codeMapBuilding"
+import { BuildingSelectedEventSubscriber, ThreeSceneService } from "../../codeMap/threeViewer/threeSceneService"
+import { Node } from "../../../codeCharta.model"
+
+class CcNodePathController implements BuildingSelectedEventSubscriber {
```

Comment:
The components/controllers are already placed within .../app/codeCharta/ui/....
Hence, I am not really convinced to prefix the Angular Components as well. It would be duplication, right?